src/mcts/params.cc

```diff
@@ -360,18 +360,10 @@ const OptionId SearchParams::kMaxConcurrentSearchersId{
     "max-concurrent-searchers", "MaxConcurrentSearchers",
     "If not 0, at most this many search workers can be gathering minibatches "
     "at once."};
-const OptionId SearchParams::kDrawScoreSidetomoveId{
-    "draw-score-sidetomove", "DrawScoreSideToMove",
-    "Score of a drawn game, as seen by a player making the move."};
-const OptionId SearchParams::kDrawScoreOpponentId{
-    "draw-score-opponent", "DrawScoreOpponent",
-    "Score of a drawn game, as seen by the opponent."};
-const OptionId SearchParams::kDrawScoreWhiteId{
-    "draw-score-white", "DrawScoreWhite",
-    "Adjustment, added to a draw score of a white player."};
-const OptionId SearchParams::kDrawScoreBlackId{
-    "draw-score-black", "DrawScoreBlack",
-    "Adjustment, added to a draw score of a black player."};
+const OptionId SearchParams::kDrawScoreId{
+    "draw-score", "DrawScore",
+    "Adjustment of the draw score from white's perspective. For Armageddon, "
+    "use -100."};
```

Comment:
I remember some discussion that somewhat more modest values may be better. Maybe this needs to be reworded to hint at this?

Reply:
The discussion I remember is without WDL adjustment (softening/contempt), where the issue is that from startpos, optimizing pure W is a serious/desperate uphill battle due to W starting out at 25%. Ideally we could come up with reasonable testing ideas to determine whether a more moderate value gives better/more realistic performance, but my intuition is that this was only due to pure WDL having so low white chances to begin with.

Reply:
Isn't this still true for default settings? I which case maybe we could `add a minimal suggestion for other setting to change? Something like "... and for best results adjust XXX and YYY for the opponent."

Reply:
Yep, you're right. Maybe rewording this into `value -1 gives Armageddon scoring` is better to make it less of a "this is sufficient for Armageddon performance" comment? Ultimately, I think we should deliver example configs with the Lc0 releases, and Armageddon mode would be a prime example for that.